libs/core/execution_base/include/hpx/execution_base/operation_state.hpp

```diff
@@ -45,8 +45,12 @@ namespace hpx { namespace execution { namespace experimental {
     template <typename O>
     struct is_operation_state;
 
-    HPX_INLINE_CONSTEXPR_VARIABLE struct start_t
-      : hpx::functional::tag_priority_noexcept<start_t>
+#if defined(HPX_COMPUTE_DEVICE_CODE)
+    HPX_DEVICE static const
+#else
+    HPX_INLINE_CONSTEXPR_VARIABLE
+#endif
```

Comment:
Should we hide this difference in the `HPX_INLINE_CONSTEXPR_VARIABLE` macro itself?